packages/workbox-routing/test/sw/router.js

```diff
@@ -73,4 +49,32 @@ describe(`Test of the Router class`, function() {
     expect(router._routes.get('GET')).to.have.members([getRoute1]);
     expect(router._routes.get('PUT')).to.have.members([putRoute1]);
   });
+
+  it(`should call self.addEventListener('fetch') when addFetchListener() is called`, function() {
+    const stub = sinon.stub(self.__proto__.__proto__.__proto__, 'addEventListener');
```

Comment:
I've added a comment explaining what's going on. If there's a better approach for stubbing out `self.addEventListener()` without triggering a `mocha` global leak warning, I'm definitely open, as I had to bang my head for a while before I was able to get even that to work.